integrations/visual-website-optimizer/test/index.test.js

```diff
@@ -209,6 +214,35 @@ describe('Visual Website Optimizer', function() {
         done();
       });
     });
+
+    it('should not send experiments if experiment type is not expected', function(done) {
```

Comment:
Let's also add a new test when the new setting is enabled or disabled

Reply:
Do you think we can create a test for isSPA where we check the `<head>` tag and look for the url with the script including `is_spa=1`? Like described here: https://help.vwo.com/hc/en-us/articles/360020919254

Not sure how doable is it.

Reply:
for testing `is_spa=1` inside head tag we might need to introduce enzyme for shallow rendering but doing this we might need some UI component to be in exist to load that which i think not possible.

Reply:
Agree.